Resources/ServerInfo/Rules.txt

```diff
@@ -57,18 +62,19 @@
+	• Civilian and Security staff are expected to maintain a minimum of competency and performance on their jobs, following the Standard Operating Procedure as required.
 	• Assets seized from criminals may only be used to make victims whole and to pay for any fines under the law. Any excess financials received from the sale of criminal assets is to be returned to their original owners, and are not to be plundered or extorted by security forces. This is a strict zero-tolerance policy.
 	• All players that are hired on as crew to the station as security or service staff, or through other direct jobs working for paychecks from the station, are considered non-antags and official Nanotrasen employees. Engaging in any hostile antagonistic activities, or abuse of the equipment provided while working in these roles, is strictly prohibited.
 
 [color=#a4885c][head=3]10. Sector Command roles such as Station Rep and Sheriff are vital to the operation of the sector and are held to a much higher degree of roleplay and role handling.[/head][/color]
 	• Sector Command is expected and [color=#ff0000]required [/color] to maintain pay for their respective staff as a part of their job. Payroll should be a minimum of 10,000 for station crew, 20,000 for Station Rep and Sheriff, and 15,000 for security per hour. Embezzlement of station funds is strictly forbidden.
 	• Both of these roles have higher access to station resources than should be allowed for most crew members. As such, if you must ever end your shift early, always bring your ID with you, and always return to cryosleep so that your access cannot be stolen and the station funds embezzled. If for any reason you cannot (disconnect, crash, power outage, etc.) Please ping the game admins in Discord and we will handle your in-game character accordingly.
 	• Sector Command is permitted and expected to issue and enforce any docking fines related to loitering in the terminal for shuttles that have been idle for more than 10 minutes in the station docks without a valid reason, up to and including forced eviction of shuttles.  Fines collected in this way are permitted to be used as payroll or the acquisition of station supplies. Time spent docked and waiting at the station is to be kept at a minimum.
-	• You may not tax, lease, rent, or otherwise reserve the free and fair usage of station resources to the detriment of the public. That includes but not limited to, charging docking fees, basic cloning services, vending access, and moving station resources behind access locked doors to restrict access, etc.
+	• You may not tax, lease, rent, or otherwise reserve the free and fair usage of station resources and space to the detriment of the public or for personal gain. That includes but not limited to, charging docking fees, granting docking extensions, vending access, and moving station resources behind access locked doors to restrict access, etc.
```

Comment:
Minor changes.
```suggestion
	• You may not tax, lease, rent, or otherwise reserve the free and fair use of station resources and space to the detriment of the public or for personal gain. This includes, but is not limited to: charging docking fees, granting docking extensions, vending access, and moving station resources behind access locked doors to restrict access.
```